docker-compose.yml

```diff
@@ -72,8 +72,6 @@ services:
       lagoon.type: cli-persistent
       lagoon.persistent.name: nginx # mount the persistent storage of nginx into this container
       lagoon.persistent: /app/web/sites/default/files/ # location where the persistent storage should be mounted
-    environment:
-      << : *default-environment # loads the defined environment variables from the top
```

Comment:
Why this change in the docker setup... don't think it should be altered.

Reply:
Did not notice when i reviewed the files, I think this is a leftover from Ture. I'll revert

Reply:
Checked it out, it seems "environment" is repeated 
https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/develop/docker-compose.yml#L66-L76